tests/headers/use-core.h

```diff
@@ -0,0 +1,6 @@
+// bindgen-flags: --use-core --raw-line "extern crate core;"
```

Comment:
Nope, at least not completely automatically I guess.

It's not uncommon doing `include!("path/to/bindings.rs")`, in which case it's unnecessary. In general it's completely unnecessary as long as the bindings aren't the crate root, so I went the easy/safe way. if it's harmless we can do it in a followup, in which case is straight-forward to do.